styles/styles.css

```diff
@@ -0,0 +1,167 @@
+a:visited {
+    color:#800020;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    background-color:lime;
+
+}
+
+.about {
+    background-color:honeydew;
+    position: relative;
+    font-family:Arial, Helvetica, sans-serif ;
+    font-size:1em;
+}
+
+.background {
+    margin:auto 3em;
+}
+
+.blogs {
+    background-color:white;
+    height: 200px;
+    padding-bottom:70px;
+    margin-left:10px;
+}
+
+body{
+    font-family:Arial, Helvetica, sans-serif;
+    margin:auto 0;
+    
+}
+
+.computer-pic {
+    align-self: center;
+
+}
+
+.content {
+    background-color:cornsilk;
+    font-family:cursive;
+    font-size:1.5em;
+    color:darkslateblue;
+    height:280px;
+    padding-top: 20px;
+    text-align:center;
+    padding-bottom:20px;
+}
+
+.experience {
+    display:grid;
+    grid-template-columns:30% 70%;
+    align-items: grid-start;
+    grid-gap: 20px;
+    margin-left: 20px; 
+
+}
+
+.flex-header li a:visited {
+    color:white;
+}
+
+.flex-header{
+    padding-inline-start: 0;
+    width: 100%;
+    display: flex;
+    font-size:1.5em;
+    text-decoration: none;
+    justify-content:center; 
+}
+
+.flex-header li {
+    list-style: none;
+    padding: 0.75em;
+    
+}
+
+footer {
+    background-color:black;
+    color:white;
+    font-size: 0.sem;
+    position:absolute;
+    bottom: 0;
+    text-align:center;
```

Comment:
`center` is not a valid value

` text-align: left|right|center|justify|initial|inherit;`